text/2019-11-13-follower-replication.md

```diff
@@ -30,9 +32,9 @@ The leader must be easy to know whether or not nodes belong to the same group or
 
 As the `Groups` is configured, the leader is able to choose a group member as a *delegate* to send entries to one or the rest group members in Log Replication, which is called Follower Replication. To implement Follower Replication, we basically need three main steps:
 
-1. The leader picks a group member as a delegate of the group
-2. The leader prepares and sends some *commission*s that indicate what entries the picked delegate will send to others by a new message type MsgBroadcast.
-3. The delegate receives this message and executes the commissions
+1. The leader picks a group member as a delegate of the group.
+2. The leader tells the delegate the name list of other peers in the same group with the delegate.
```

Comment:
When processing Log Replication to a delegate, the leader must specify peers whom the delegate should replicate logs to.